bears/hypertext/HTMLLintBear.py

```diff
@@ -28,9 +28,101 @@ class HTMLLintBear:
+                         enable_protocol: bool=True,
+                         enable_names: bool=True,
+                         enable_capitalization: bool=True,
+                         enable_quotation: bool=True,
+                         enable_indentation: bool=True,
+                         enable_formatting: bool=True,
+                         enable_boolean_attribute: bool=True,
+                         enable_invalid_attribute: bool=True,
+                         enable_void_zero: bool=True,
+                         enable_invalid_handler: bool=True,
+                         enable_http_equiv: bool=True,
+                         enable_extra_whitespace: bool=True,
                          htmllint_ignore: typed_list(str)=()):
         """
-        :param htmllint_ignore: List of checkers to ignore.
+        :param enable_doctype:
+            Enable or disable doctype checker.
+        :param enable_entities:
+            Enable or disable unicode entity checker.
+        :param enable_trailing_whitespace:
+            Enable or disable checker for trailing whitespace.
+        :param enable_tabs:
+            Enable or disable check for tabs.
+        :param enable_charset:
+            Enable or disable utf-8 checking.
+        :param enable_void_element:
+            Enable or disable checker for void tags.
+        :param enable_optional_tag:
+            Enable or disable checker for optional tags.
+        :param enable_type_attribute:
+            Enable or disable checker for default types.
+        :param enable_concerns_separation:
+            Enable or disable checker for seperation of concern.
+        :param enable_protocol:
+            Enable or disable checker for protocol specification.
+        :param enable_names:
+            Enable or disable checker for id or class name delimiters.
+        :param enable_capitalization:
+            Enable or disable checker for capitalization
+            of tags and attributes.
+        :param enable_quotation:
+            Enable or disable checker for quotation marks.
+        :param enable_indentation:
+            Enable or disable checker for indentation using spaces.
+        :param enable_formatting:
+            Enable or disable checker for general formatting.
+        :param enable_boolean_attribute:
+            Enable or disable checker for boolean attributes.
+        :param enable_invalid_attribute:
+             Enable or disable checker for invalid attributes.
+        :param enable_void_zero:
+           Enable or disable checker for javascript void(0).
+        :param enable_invalid_handler:
+           Enable or disable checker for Javascript links.
+        :param enable_http_equiv:
+           Enable or disable checker for http-equiv.
+        :param enable_extra_whitespace:
+           Enable or disable checker for extra whitespace.
+        :param htmllint_ignore:
+           List of checkers to ignore.
```

Comment:
i feel like, you should instead tell what the setting do instead of telling what is enabled or disabled